src/cli/MareTF.cpp

```diff
@@ -415,6 +686,68 @@ int main(int argc, const char* const argv[]) {
 		.flag()
 		.store_into(hdriNoFilter);
 
+	bool alphaToDistance;
+	createCLI
+		.add_argument("--alpha-to-distance")
+		.help("Convert alpha channel to a distance map.")
+		.flag()
+		.store_into(alphaToDistance);
+
+	uint16_t reduce = 1;
+	createCLI
+		.add_argument("--reduce")
+		.metavar("FACTOR")
+		.help("Factor by which to downscale when building a distance map."
+			" Must be a power of 2. Overridden by --reduce-x and --reduce-y.")
+		.scan<'u', uint16_t>()
+		.default_value(reduce).store_into(reduce);
+
+	bool reduceXSet = false;
+	uint16_t reduceX = 1;
+	createCLI
+		.add_argument("--reduce-x")
+		.metavar("FACTOR")
+		.action([&](const auto&){ reduceXSet = true; })
+		.help("Factor by which to downscale width when building a distance map."
+			" Must be a power of 2.")
+		.scan<'u', uint16_t>()
+		.default_value(reduceX).store_into(reduceX);
+
+	bool reduceYSet = false;
+	uint16_t reduceY = 1;
+	createCLI
+		.add_argument("--reduce-y")
+		.metavar("FACTOR")
+		.action([&](const auto&){ reduceYSet = true; })
+		.help("Factor by which to downscale height when building a distance map."
+			" Must be a power of 2.")
+		.scan<'u', uint16_t>()
+		.default_value(reduceX).store_into(reduceY);
+
+	float distanceSpread = 1.0f;
+	createCLI
+		.add_argument("--distance-spread")
+		.metavar("COEFF")
+		.help("Multiply the search radius when building a distance map.")
+		.scan<'g', float>()
+		.default_value(distanceSpread).store_into(distanceSpread);
+
+	bool noValveDistanceQuirks;
+	createCLI
+		.add_argument("--no-valve-distance-quirks")
+		.flag()
+		.help("Do not force the edges of a generated distance map to black, nor warn about"
+			" finite distances at the edges of an image.")
+		.store_into(noValveDistanceQuirks);
+
+	bool distanceAA;
+	createCLI
+		.add_argument("--distance-aa")
+		.flag()
+		.help("Treat any alpha channel turned into a distance map as antialiased."
+			" May cause artifacts if long gradients are present.")
```

Comment:
I know it sucks to write this with an IDE, but for help text I use spaces to align on new string lines. This way it looks nice no matter what tab size you have

```suggestion
		      " May cause artifacts if long gradients are present.")
```